### PR TITLE
MS-9 add test file

### PR DESCRIPTION
**Pull request by AashishGrg**
@SummitStha @raeesaroj @ankurbelbase @santoshpurbey

### CODE REVIEW

File: mysite/mybranch.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	contact<br>
+	places<br>
+	</div>
+		<div class="section">
+		<h1>Kathmandu</h1>
+		<p> Kathmandu is the capital city of Nepal.It is the most populous city in Nepal.</p>
+		</div>
+			<div class="footer">
+			visit us @ <a href="nepalmandu.com">nepalmandu.com</a</div>
+
```
Comment (imanilpun) [inline mysite/mybranch.txt:23]: add more section

File: mysite/mybranch.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="visitnepal.css">
+<head>
+	<title>Visit Nepal</title>
```
Comment (AlsaRb) [inline mysite/mybranch.txt]: Give title "bitsinnovation"

File: mysite/mybranch.txt (hunk 1 above)
Comment (AashishGrg) [reply mysite/mybranch.txt:23]: ok man

**Comment by AashishGrg** [comment]
ok

**Pull request merged**